salt/utils/__init__.py

```diff
@@ -1587,6 +1587,40 @@ def is_sunos():
 
 
 @real_memoize
+def is_smartos():
+    '''
+    Simple function to return if host is SmartOS (Illumos) or not
+    '''
+    if not is_sunos():
+        return False
+    else:
```

Comment:
fixed in follow up commit, must have skipped that line when I was fixing the indentation.